docs/support/faq.md

```diff
@@ -158,75 +158,75 @@ For other permissions, please see the contract [here](https://github.com/LIT-Pro
+**(cont'd) It says that it uses BLS threshold signatures so that the decryption key is split into multiple pieces, but that doesn't really totally explain how the key management works? How do nodes learn about the various keys being managed by the network? How do you prevent one node operator from accumulating all the component keys needed to reconstruct any of the symmetric decryption keys?**
 
-Each node only holds a private key share. When a user wants to decrypt something, he presents the thing to decrypt, and proof that he meets the conditions (a wallet signature). Each node independently checks that the user meets the condition with a RPC call to an ETH node. If the user meets the condition, the node uses its private key share to create a decryption share. The user collects the decryption shares and accumulates them above the threshold, and is then able to decrypt the content.
+Each node only holds a private key share. When a user wants to decrypt something, they present the thing to decrypt and proof that they meet the conditions (a wallet signature). Each node independently checks that the user meets the condition with an RPC call to an ETH node. If the user meets the condition, the node uses its private key share to create a decryption share. The user collects the decryption shares and accumulates them above the threshold and is then able to decrypt the content.
 
 So, you can see, the nodes don't talk to each other when decrypting the content. Each node's private key share never leaves the node.
 
 ### 4. How do new nodes that come online discover the key shares they need to help decrypt previously-encrypted data? Or is each key fragment tied to a single node lifetime?
 
 **(cont'd) If the latter, then doesn't that mean that there's no redundancy on the key fragments and you're very susceptible to nodes going offline? Like if too many nodes go offline then you might no longer have enough key fragments in the online nodes to decrypt some pieces of content?**
 
-Right now, we're running all the nodes, so the nodes and shares don't change. soon, we will run a federated network with named nodes, and the ultimate goal is to run a permissionless one. we use a process called proactive secret sharing to share the private key shares with new nodes as they come online. the shares given to new nodes are incompatible with any nodes that have left the network. we use threshold encryption with a 2/3 threshold so redundancy is built in.
+Right now, we're running all the nodes, so the nodes and shares don't change. Soon, we will run a federated network with named nodes, and the ultimate goal is to run a permissionless one. We use a process called proactive secret sharing to share the private key shares with new nodes as they come online. The shares given to new nodes are incompatible with any nodes that have left the network. We use threshold encryption with a 2/3 threshold, so redundancy is built-in.
```

Comment:
this needs to be updated, as Manzano and Habanero are decentralized networks being run by 3rd party operators. We are only running the nodes for the other networks, which we should specify here

Reply:
Added @a1ttech

Reply:
added @a1ttech